### PR TITLE
feat: added datadog logging adapter

### PR DESCRIPTION
**Pull request by abdullahwaheed**
**Description:**

Describe what this pull request changes, and why. Include implications for people using this change.

**Merge checklist:**

- [ ] Consider running your code modifications in the included example app within `frontend-platform`. This can be done by running `npm start` and opening http://localhost:8080.
- [ ] Consider testing your code modifications in another local micro-frontend using local aliases configured via [the `module.config.js` file in `frontend-build`](https://github.com/openedx/frontend-build#local-module-configuration-for-webpack).
- [ ] Verify your commit title/body conforms to the conventional commits format (e.g., `fix`, `feat`) and is appropriate for your code change. Consider whether your code is a breaking change, and modify your commit accordingly.

**Post merge:**

- [ ] After the build finishes for the merged commit, verify the new release has been pushed to [NPM](https://www.npmjs.com/package/@edx/frontend-platform).

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -27,4 +27,5 @@ FAVICON_URL=https://edx-cdn.org/v3/default/favicon.ico
 IGNORED_ERROR_REGEX=
 MFE_CONFIG_API_URL=
 APP_ID=
-SUPPORT_URL=https://support.edx.org
\ No newline at end of file
+SUPPORT_URL=https://support.edx.org
+DATADOG_ENABLED=false
```
Comment (schenedx) [inline .env.development:31]: Could you please keep this list alphabetized? So, insert this line into line 8?

File: package.json [hunk 2]
```diff
@@ -35,8 +35,8 @@
   "devDependencies": {
     "@edx/brand": "npm:@openedx/brand-openedx@^1.2.2",
     "@edx/browserslist-config": "1.2.0",
-    "@openedx/frontend-build": "14.0.3",
-    "@openedx/paragon": "22.3.1",
```
Comment (schenedx) [inline package.json:39]: Why do we want to lower the version of Paragon? Is it just the PR is old?

File: package.json [hunk 3]
```diff
@@ -35,8 +35,8 @@
...
+    "@openedx/frontend-build": "github:openedx/frontend-build#abdullahwaheed/datadog-logging",
```
Comment (schenedx) [inline package.json:38]: Are you planning to merge this change to master? Or just use this branch on an MFE for testing first?

**Review by schenedx — COMMENTED** [review]
See my inline questions

File: package.json (hunk 2 above)
Comment (abdullahwaheed) [reply package.json:39]: it was synced with old alpha branch, let me rebase it

File: package.json (hunk 3 above)
Comment (abdullahwaheed) [reply package.json:38]: no, we'll first merge it in alpha to test it

**Comment by adamstankiewicz** [comment]
@abdullahwaheed Is it OK to close this PR at this point?

**Comment by abdullahwaheed** [comment]
yes, this was testing PR. closing it now

**Pull request closed without merging**